pipelines/rj_crm__disparo_template/utils/dispatch.py

```
@@ -972,3 +1078,150 @@ def get_retry_destinations(
+        for local_file in [csv_path]:
+            if os.path.exists(local_file):
+                # log(f"Arquivos .csv do diretório atual antes de enviar novo arquivo: {[f for f in sftp.listdir('.') if f.endswith('.csv')]}")
+                file_name = os.path.basename(local_file)
+                log(f"Enviando {file_name}...")
+                sftp.put(local_file, file_name)
+                log(f"Arquivo {file_name} enviado com sucesso.")
+            else:
+                log(f"Arquivo não encontrado: {local_file}")
+        sftp.close()
+        
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)